src/testing/testing_runtime.ts

```diff
@@ -128,6 +129,10 @@ export class TestingRuntimeImpl implements TestingRuntime {
     return value as T;
   }
 
+  setConfig<T>(key: string, newValue: T): void {
```

Comment:
What will happen if the applicationConfig is undefined (not configured in dbos-config file), but we wanted to introduce some new configs in our tests? Should we create the applicationConfig first if it's undefined?